torch/_dynamo/variables/tensor.py

```diff
@@ -394,6 +394,13 @@ def call_obj_hasattr(self, tx: "InstructionTranslator", name):
         from . import GetAttrVariable
         from .builtin import BuiltinVariable
 
+        # TODO - This is not a good solution but solves an accuracy issue.
+        # Today, var_getattr returns GetAttrVariable for both non-existent
+        # attributes and existing attributes. This is a bug and requires more
+        # deep dive.
+        if name in ("size", "stride"):
+            return ConstantVariable(True)
```

Comment:
Without this, do we always end up getting `ret_val = False` below or something?

Reply:
Yes. See the test! Such an embarrassing failure.

Reply:
We basically go into this `ret_val = not isinstance(var, GetAttrVariable)`